app/src/lib/stores/app-store.ts

```diff
@@ -3009,14 +3009,17 @@ export class AppStore extends TypedBaseStore<IAppState> {
     repository: Repository,
     name: string,
     startPoint: string | null,
-    noTrackOption: boolean = false
-  ): Promise<void> {
+    noTrackOption: boolean = false,
+    checkoutBranch: boolean = false
```

Comment:
I couldn't see where this new parameter is used. Is this a leftover?

Reply:
Ugg.. good catch.. that should default to true..